processor/src/main/java/org/mapstruct/ap/MappingProcessor.java

```diff
@@ -5,16 +5,17 @@
  */
 package org.mapstruct.ap;
 
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ServiceLoader;
-import java.util.Set;
+import org.mapstruct.ap.internal.model.Mapper;
```

Comment:
Something strange is happening with the imports here..

Reply:
Yes sorry.  I think I have inadvertently reorganized imports for that class. I will undo it.
However, I paid attention to import your checkstyle rules in my IntelliJ. The import order layout is not defined inside and it looks like you have been using Eclipse default settings. Maybe it should be a good idea to add a chekstyle rule for that to keep cross IDE usage more consistent.

Reply:
There are still some classes that don't have the proper formatting completely applied to them. The Eclipse and IntelliJ formatter should be aligned already (unless we have missed something). Adding checkstyle might help, but we would first need to fix all classes 😄

Reply:
I had missed the `/etc` folder with formaters, I thought all style rules were in the checkstyle.xml in `/build-config` folder. 🔎🔎🔎
Thank you for rising the topic of formaters.